src/tasks/tool-installer/tool-installer-v2/index.ts

```diff
@@ -18,6 +18,7 @@ export async function main(): Promise<void> {
 
   tl.debug(`Found required pac CLI executable under: ${pacPath}`);
   tl.debug(`Setting ${PacPathEnvVarName} : ${pacPath}`);
+  tl.prependPath(pacPath);
```

Comment:
we thought doing it with optional argument for example: AddToolsToPath because this has a potential of breaking working pipelines as more tools unexpectedly show up in path

@tehcrashxor

Reply:
Will work on that.